src/rust/lib/graph-editor/src/component/cursor.rs

```diff
@@ -54,12 +64,21 @@ impl<T:Default> Default for StyleValue<T> {
 impl<T> StyleValue<T> {
     /// Constructor.
     pub fn new(value:T) -> Self {
+        let value   = Some(value);
+        let animate = true;
+        Self {value,animate}
+    }
+
+    /// Constructor.
+    pub fn new_default() -> Self {
```

Comment:
Shouldn't this be just `Default` trait implementation?

Reply:
No, it means something else. Adding more docs.